spring-boot-cli/src/main/java/org/springframework/boot/cli/command/init/InitCommand.java

```diff
@@ -96,6 +97,18 @@ public String getUsageHelp() {
 
 		private OptionSpec<Void> force;
 
+		private OptionSpec<String> language;
+
+		private OptionSpec<String> groupId;
+
+		private OptionSpec<String> artifactId;
+
+		private OptionSpec<String> name;
+
+		private OptionSpec<String> version;
+
+		private OptionSpec<String> description;
+
```

Comment:
Can we reorganize these options so that they fit a more natural order please? (not adding them "at the end")